queries.js

```diff
@@ -0,0 +1,43 @@
+const pgp = require('pg-promise')()
+const connectionString = 'postgres://localhost:5432/sweethornbillbookstore'
+const db = pgp(connectionString)
+
+
+
+const getAllBooks = () => db.any( 'SELECT * FROM books' )
+
+const getSingleBook = ( request, response ) => {
+  const id = request.params.id
+  return db.one('SELECT * FROM books WHERE id = $1', id)
+}
+
+const getSingleAuthor = ( request, response ) => {
+  const author = request.params.author.toLowerCase()
+    console.log(author)
+    return db.one('SELECT * FROM books WHERE LOWER(author) = $1', author)
+}
+
+const getSingleGenre = ( request, response ) => {
+  const genre = request.params.genre
+    return db.one('SELECT * FROM books WHERE genre = $1', genre)
+}
+
+const addBook = ( request, response ) => {
+  return db.any('SELECT * FROM books')
+}
+
+const createBook = ( request, response ) => {
+  const { title, author, genre, description, img_url } = request.body
```

Comment:
what i did was create an addBookPage so that i can make a form to add a book, then i have the createBook page that does all the sending of the info on the form as well as redirects to the newly created book. do you have any suggestions on how to simplify this?